resources/measures/upgrade_df_thermostat_control_load_shed/measure.rb

```diff
@@ -371,19 +438,37 @@ def applicable_thermostats(model)
       return applicable_clg_thermostats, applicable_htg_thermostats, thermostats.size
     end
 
+    # adding output variables (for debugging)
+    out_vars = [
+      'Lights Electricity Energy',
+      'Lights Electricity Rate',
+      'Zone Lights Electricity Energy',
+      'Zone Lights Electricity Rate',
+      'Schedule Value'
+    ]
+    out_vars.each do |out_var_name|
+        ov = OpenStudio::Model::OutputVariable.new('ov', model)
+        ov.setKeyValue('*')
+        ov.setReportingFrequency('timestep')
+        ov.setVariableName(out_var_name)
+    end
+
     ############################################
     # Applicability check
     ############################################
     puts("### ============================================================")
     puts("### Applicability check...")
     applicable_building_types = [
       # "Hotel",
-      # "SmallOffice",
-      # "MediumOffice",
+      "SmallOffice",
+      "MediumOffice",
       "LargeOffice",
       "OfL",
       "OfS",
-      "Office"
+      "Office",
+      "Warehouse",
+      "PrimarySchool",
+      "SecondarySchool"
```

Comment:
@JieXiong9119 don't we have other building type names for warehouse/school in CA? similar to OfL/OfS?

Reply:
maybe this: https://github.com/NREL/ComStock/blob/1f5866c4dd9a499df2523fc20f5a017a3349036b/resources/measures/set_hvac_template/resources/deer_building_types.rb#L10-L36